src/components/Comment/index.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import commentsData from "../../data/comments";
+import CommentElement from "./CommentElement";
+
+const Comment = ({ postId }) => {
+  // 댓글들을 저장하기 위한 state
+  const [comments, setComments] = useState(commentsData);
+  // 새로운 댓글을 추가하기 위한 state
+  const [newComment, setNewComment] = useState("");
+
+  const handleCommentSubmit = (e) => {
+    e.preventDefault();
+    // 현재 날짜와 시간을 포함하는 새 댓글 객체 생성
+    const now = new Date();
+    const createdAt = now.toISOString().split("T")[0];
+
+    const commentToAdd = {
+      id: comments.length + 1,
+      content: newComment,
+      created_at: createdAt,
+      author: { id: "1", username: "currentUser" },
+    };
+    setComments([...comments, commentToAdd]);
+    setNewComment("");
+    alert("댓글 작성");
+  };
+
+  const handleCommentEdit = (id, editedContent) => {
+    setComments(
+      comments.map((comment) => {
+        if (comment.id === id) {
+          return { ...comment, content: editedContent };
+        }
+        return comment;
+      })
+    );
+  };
+
+  const handleCommentDelete = (commentId) => {
+    const updatedComments = comments.filter(
+      (comment) => comment.id !== commentId
+    );
+    setComments(updatedComments);
+    alert("댓글 삭제");
+  };
+
+  return (
+    <div className="w-full mt-5 self-start">
+      <h1 className="text-3xl font-bold my-5">Comments</h1>
+      {comments.map((comment) => (
+        <CommentElement
+          key={comment.id}
+          id={comment.id}
+          content={comment.content}
+          createdAt={comment.created_at}
+          postId={comment.post}
+          author={comment.author}
+          handleCommentDelete={handleCommentDelete}
+          handleCommentEdit={handleCommentEdit}
```

Comment:
```suggestion
          comment={comment}
          handleCommentDelete={handleCommentDelete}
          handleCommentEdit={handleCommentEdit}
```
작성해준 방식으로 props를 넘겨줄 수도 있지만 `comment` 객체를 넘겨주는 건 어떨까요? 일일이 props를 적기에도 번거롭고 `CommentElement`에서 지금 모든 props를 활용하고 있지 않으니까 `comment` 자체를 넘겨주고 `CommentElement`에서 필요한 내용을 디스트럭처링 할당으로 활용하는 것도 좋을 것 같아요!